internal/template/funcmap.go

```diff
@@ -72,21 +72,17 @@ func hcl(v interface{}) (string, error) {
 	return string(b), nil
 }
 
-// hclField returns a hcl marshaled field e.g. `name = "foo"`
-func hclField(m map[string]interface{}, key string, req bool) (string, error) {
+// hclField returns a hcl marshaled field e.g. `name = "foo"`, if present.
```

Comment:
maybe add a note if want to enforce the field to be present, use `hcl`

Reply:
Good idea, done